pkgs/desktops/cinnamon/nemo-extensions/srcs.nix

```diff
@@ -1,15 +0,0 @@
-{ fetchFromGitHub }:
-
-rec {
-  # When you bump this, you should make sure all nemo-extensions
-  # are actually using this file since we try to deal with tags
-  # like nemo-fileroller-5.6.1 according to upstream's wishes.
```

Comment:
What wishes are those?

Reply:
nemo extensions use the same [mono repo](https://github.com/linuxmint/nemo-extensions/) but does not always uses the same tag, example is nemo-emblems 6.2.1 in http://packages.linuxmint.com/search.php?release=wilma&section=any&keyword=nemo